torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -2696,18 +2726,17 @@ def _register_pre_forward_hooks(self):
         for forward_handle in self._pre_forward_handles:
             forward_handle.remove()
         self._pre_forward_handles.clear()
+
```

Comment:
nit: intentional newline? (since `_register_post_forward_hooks()` does not have)

Reply:
removed this line